.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.json]
```

Comment:
I don't mind removing this, or filling it out more (e.g. with *.py or *.toml configuration)

Reply:
Let's leave it as-is, editorconfig is neat!